automated-comments/go/hamming/distance_signature_changed.md

```diff
@@ -0,0 +1,4 @@
+The signature of the `Distance` function should be `Distance(a, b string) (int, error)`.
+Exercism is built on the TDD (Test Driven Development) principle.
+The tests already exist and define the outward facing interface as well as the behaviour of the code to be implemented.
+Changing the signature will break the tests.
```

Comment:
```suggestion
Changing the signature breaks the tests.
```

Reply:
Same comment here about potentially converting this to a general comment with parameters.

Reply:
Will do that in the future. The current analyzer that is online already provides these comments on disapprove, so I am trying to get them in there without a new release for now.